frontend/src/components/signUpWindow.jsx

```diff
@@ -89,6 +91,7 @@ function SignUpWindow(props) {
                 onChange={setConfirmPassword}
                 mask
             />
+            <InputBox placeholder="Biography" onChange={setDescription} />
```

Comment:
I think this should be a textbox field (like the description field in the upload window)